Shrine/app/src/main/java/com/authentication/shrine/api/AuthApiService.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 The Android Open Source Project
```

Comment:
why 2024 copyright? Didnt we run spotless to uto add or this is copy pasted? Can we correct this?